src/Firebase/Auth/UserMetaData.php

```diff
@@ -22,7 +22,7 @@ class UserMetaData implements \JsonSerializable
     public static function fromResponseData(array $data): self
     {
         $metadata = new self();
-        $metadata->createdAt = DT::toUTCDateTimeImmutable($data['createdAt']);
+        $metadata->createdAt = isset($data['createdAt']) ? DT::toUTCDateTimeImmutable($data['createdAt']) : null;
```

Comment:
I fixed it, but not really needed right now because we use the `getUser` endpoint everywhere now which always returns the `createdAt` field, but I think it may be handy in the future.

Reply:
I perfectly understand your intent, but if we don't need it now, I would feel more comfortable not changing it until we actually need it. This has the benefit that there's one class less to change and one test case less to cover 😅